### PR TITLE
Docs: refactors config panels and visualizations, corrects relrefs

### PR DESCRIPTION
**Pull request by chri2547**
This PR completes the refactoring of dashboard, panel, and visualization content.

### CODE REVIEW

**Comment by torkelo** [comment]
Oh boy, a lot to do here (mainly commenting on the state of these docs and how poor or out of date they are). 

Some structrual issues
* Managing library panels should be moved to `Dashboards and visualizations" root
 
**Add and organize panels** this page needs a major update. It currently covers the panel edit UI (in a bad way), but this should belong to the Panels and visualizations root topic I think. The add and organize panels should be about adding panels to a dashboard, how to move and resize panels and how you can add rows to a dashboard that give grouops of panels a title and a way to collapse them. 

**Use dashboard** currently has a section on search. We should move this to new root topic (Ie content root). Search is and will be a more global thing. If we have global search topic we can then move Dashboard previews topic under the Search category as it's an experimental feature only relevant in the search view. 

the Manage dashbords page has some content that is part of using dashboards, the section on `Configure dashboard time range controls`  (better named Set dashboard time range). 

There is also a section in manage dashboards for `Dashboard time settings` that should probably be part of the build dashboards > Dashboard settings page (which we are missing) 

**Panel time overrides and timeshift** think should go into a new page under Configure panels and visualizations -> Query options  

Feels a bit overwhelming looking at this, but I can help with some of this if I can find some time to spend on it. 

Another thought after testing this a bit is that the nesting feels a bit too much when you come down to visualization types. Maybe we can keep Panels and visualizations as a root category (as a sibling to Dashboards) (like it it is today). I think panels and visualizations will be something that we see in not only dashboards in the future so could be good to position them as more standalone building blocks.

**Comment by chri2547** [comment]
> 

pushed a series of commits that address each point you made.

**Comment by chri2547** [comment]
> Oh boy, a lot to do here (mainly commenting on the state of these docs and how poor or out of date they are).
> 
> **Add and organize panels** this page needs a major update. It currently covers the panel edit UI (in a bad way), but this should belong to the Panels and visualizations root topic I think. The add and organize panels should be about adding panels to a dashboard, how to move and resize panels and how you can add rows to a dashboard that give groups of panels a title and a way to collapse them.
> 
> **Use dashboard** currently has a section on search. We should move this to new root topic (Ie content root). Search is and will be a more global thing. If we have global search topic we can then move Dashboard previews topic under the Search category as it's an experimental feature only relevant in the search view.
> 
> the Manage dashbords page has some content that is part of using dashboards, the section on `Configure dashboard time range controls` (better named Set dashboard time range).
> 
> There is also a section in manage dashboards for `Dashboard time settings` that should probably be part of the build dashboards > Dashboard settings page (which we are missing)
> 
> **Panel time overrides and timeshift** think should go into a new page under Configure panels and visualizations -> Query options
> 
> Feels a bit overwhelming looking at this, but I can help with some of this if I can find some time to spend on it.
> 
> Another thought after testing this a bit is that the nesting feels a bit too much when you come down to visualization types. Maybe we can keep Panels and visualizations as a root category (as a sibling to Dashboards) (like it it is today). I think panels and visualizations will be something that we see in not only dashboards in the future so could be good to position them as more standalone building blocks.

pushed a series of commits that address each point above.

**Comment by torkelo** [comment]
@chri2547 think this is getting close. Moved some topics that related to dashboard building from the "Add and organize panels" to Create dashboard topic, and the remaining became a "Panel editor overview" topic under panels. 

Changed the ordering a bit so that Dashboards is above panels, in menu.

Moved some more topics from the left over "Panels" topic. The only thing remaining there is the Query a data source which I think we should  move to root under the name "Data queries"  or just "Queries". I don't think we can merge this (at least not backport it) in the current state with the old panels topic remaining and with only this  as child topic. 

The other things that folder naming "configure-panels-and-visualizations" is unnecessarily long and complex. Prefer to keep folder names as plain and simple, to the point as possible. "panels"  or "panels-and-visualizations"  work better and will makes refs a bit more easy to read.

File: docs/sources/datasources/prometheus.md [hunk 1]
```diff
@@ -92,13 +92,13 @@ The "Validate selector" button will check with Prometheus how many time series a
...
+| `Type`      | `Range` - Query returning a Range vector, a set of time series containing a range of data points over time for each time series.<br/>`Instant` - Perform an "instant" query to return only the latest value that Prometheus has scraped for the requested time series. Instant queries can return results much faster than normal range queries. Use them to look up label sets. Instant query results are made up only of one data point per series but can be shown in the graph panel in a dashboard with the help of [series overrides]({{< relref "../panels-visualizations/visualizations/graph/#series-overrides" >}}). To show them in the graph as a latest value point, add a series override and select `Points > true`. To show a horizontal line across the whole graph, add a series override and select `Transform > constant`. <br/>`Both` - Available only in Explore. Runs both range and instant query |
```
Comment (chri2547) [inline docs/sources/datasources/prometheus.md:100]: @torkelo , this line introduces some instructions that related to the graph panel, which we've removed. I was unsure of how to adjust this content.

Comment (torkelo) [reply docs/sources/datasources/prometheus.md:100]: @chri2547 the same feature can be accomplished adding a field override, and add property to the override named "Transfrom" and selecting Constant from the Transform dropdown 
https://grafana.com/docs/grafana/latest/visualizations/time-series/#transform

**Review by torkelo — APPROVED** [review]

**Review by torkelo — APPROVED** [review]

**Pull request merged**